### ISSUE TITLE
Use test_case

### ISSUE THREAD

**Issue by ThomasTr** [symfony/panther#95]
Is it possible to get the webserver with a test_case started? The page I want to test uses vue.js and the controller normaly requests an external API, which I would like to mock.

**Comment by dunglas** [comment]
You mean using a custom [Symfony environment](https://symfony.com/doc/current/configuration/environments.html)?

When https://github.com/symfony/recipes/pull/464 will be merged, you'll be able to just run `APP_ENV=myenv phpunit` to do it, and it will default to `test` (currently `dev` is used).

**Comment by ThomasTr** [comment]
Not really. In my tests i have a lot of different test_case as also done in symfony functional tests:
```
$client = $this->createClient(array('test_case' => 'ContainerDump', 'root_config' => 'config.yml'));
```
And then testdoubles in which I overwrite e.g. the method of a service that an external API is not called and instead dummy code is returned.

**Comment by dunglas** [comment]
Fixed by #99.

**Comment by ThomasTr** [comment]
Thanks @dunglas 👍
